src/pyhf/pdf.py

```diff
@@ -176,9 +178,7 @@ def _create_and_register_paramsets(
 
 
 class _ConstraintModel(object):
-    """
-    Factory class to create pdfs for the constraint terms
-    """
+    """Factory class to create pdfs for the constraint terms."""
```

Comment:
This PR is also a good motivation for us to formalize our docstring style. Just as one example here, do we want to say that as our style we do

```
"""
Description line of class or method
"""
```

or

```
"""Description line of class or method"""
```
?

Reply:
Ah okay, that greatly simplifies things. Thanks!